airbyte-integrations/connectors/source-bigquery/Dockerfile

```diff
@@ -9,5 +9,5 @@ COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 RUN tar xf ${APPLICATION}.tar --strip-components=1
 
 # Airbyte's build system uses these labels to know what to name and tag the docker images produced by this Dockerfile.
-LABEL io.airbyte.version=0.1.1
+LABEL io.airbyte.version=0.1.2
 LABEL io.airbyte.name=airbyte/source-bigquery
```

Comment:
nit: I know this wasn't checked in by you, but can you add a new line here. thanks

Reply:
Sure, done